modules/exploits/multi/http/osCommerce.rb

```diff
@@ -0,0 +1,69 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
```

Comment:
Is there a reason it's not `ExcellentRanking` ?

As per https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking : 

> ExcellentRanking -	The exploit will never crash the service. This is the case for SQL Injection, CMD execution, RFI, LFI, etc. No typical memory corruption exploits should be given this ranking unless there are extraordinary circumstances (WMF Escape()).